setup.py

```diff
@@ -30,5 +34,5 @@ def read(fname):
     classifiers=[
         "Development Status :: 0 - Alpha",
     ],
-    packages=['whylog'],
+    packages=['whylog'] + ['whylog.' + subpackage for subpackage in subpackages],
```

Comment:
the standard way is to use https://pythonhosted.org/setuptools/setuptools.html#using-find-packages